ext/drsyscall/drsyscall_record.h

```diff
@@ -90,7 +90,7 @@ typedef enum {
  * value.
  */
 START_PACKED_STRUCTURE
-typedef struct syscall_record_t_ {
+typedef struct ALIGN_VAR(sizeof(reg_t)) syscall_record_t_ {
```

Comment:
Looks like this type is currently 18 bytes in size for x64: so this change adds 6 bytes of padding between every record. That's a lot: 33% increase. Trying to remember how this works: there are a bunch of syscall_record_t for each syscall with buffer data mixed in there: so it's not like there's this padding more than once for a single syscall buffer? What is the size increase in your tests?